core/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-all.zip
```

Comment:
should be in 7.6.3

Reply:
Its 7.1.1 in main. Should I target a different branch for the PR?

Reply:
I'v switched to 3.5.4. But Branch 3.6 brings Gradle 7.6